src/ios/MixpanelPlugin.h

```diff
@@ -34,7 +34,6 @@
 -(void)people_append:(CDVInvokedUrlCommand*)command;
 -(void)people_deleteUser:(CDVInvokedUrlCommand*)command;
 -(void)people_increment:(CDVInvokedUrlCommand*)command;
--(void)people_setPushId:(CDVInvokedUrlCommand*)command;
```

Comment:
this removal (by mixpanel lib) might be a problem.
the cordova plugin currently exposes `window.mixpanel.people.setPushId` and expects it to be available for both android & ios :( .

we'll need to adapt `setPushId` a little so existing code that uses this on ios doesn't break suddenly.

dya know of any documentation saying why was this removed? maybe was replaced with something else?